crates/miden-objects/src/block/block_inputs.rs

```diff
@@ -0,0 +1,120 @@
+use std::collections::BTreeMap;
+
+use crate::{
+    account::AccountId,
+    block::BlockHeader,
+    crypto::merkle::{MerklePath, SmtProof},
+    note::{NoteId, NoteInclusionProof, Nullifier},
+    transaction::ChainMmr,
+    Digest,
+};
+
+// BLOCK INPUTS
+// ================================================================================================
+
+/// Information needed from the store to build a block
+#[derive(Clone, Debug)]
+pub struct BlockInputs {
+    /// Previous block header
+    prev_block_header: BlockHeader,
+
+    /// MMR peaks for the current chain state
+    chain_mmr: ChainMmr,
```

Comment:
Is this just MMR peaks, or does it also contain authentication paths for all batch reference blocks?

Reply:
It's also authentication paths. The doc comment was outdated.